Client/Frontend/Contextual Hint/ContextualHintEligibilityUtility.swift

```diff
@@ -52,8 +52,7 @@ struct ContextualHintEligibilityUtility: ContextualHintEligibilityUtilityProtoco
     /// - the JumpBackInSyncedTab CFR has NOT been presented already
     /// - the JumpBackIn CFR has NOT been presented yet
     private var canJumpBackInBePresented: Bool {
-        guard UIDevice.current.userInterfaceIdiom != .pad,
```

Comment:
Are we fully removing the iPad check, or is this just checked somewhere else?

Reply:
Removing it fully!